io-engine/src/bdev/nexus/mod.rs

```diff
@@ -123,12 +123,17 @@ pub fn register_module() {
 
 /// called during shutdown so that all nexus children are in Destroying state
 /// so that a possible remove event from SPDK also results in bdev removal
-pub async fn nexus_children_to_destroying_state() {
-    info!("Setting all nexus children to destroying state...");
-    for nexus in nexus_iter() {
-        for child in nexus.children_iter() {
-            child.set_state(nexus_child::ChildState::Destroying);
+pub async fn shutdown_nexuses() {
```

Comment:
s/shutdown_nexuses/destroy_nexuses ?
Just to distinguish this from the shutdown on your other PR?